docusaurus/docs/learn/quickstarts/services/kubernetes-service.md

```diff
@@ -2,69 +2,81 @@
 title: Kubernetes Service
 ---
 
-This is a quick example for tunneling to a Kubernetes workload with OpenZiti that builds on [the local Kubernetes quickstart](/learn/quickstarts/network/local-kubernetes.md). There are two deployments for you to deploy in the same namespace:
+This is a tutorial for tunneling a Kubernetes workload with OpenZiti that creates:
 
-* a demo workload service `hello-toy`
+* a demo service `hello-toy`
 * an OpenZiti reverse proxy pod `ziti-host`
 
-1. Create OpenZiti configs, service, and policies for the Hello demo deployment.
+## Steps
+
+1. You must have a OpenZiti router in the cluster with a tunnel binding. You can use this command to find eligible routers.
 
     ```text
-    ziti edge create identity device "hello-host" \
-        --jwt-output-file /tmp/hello-host.jwt --role-attributes hello-hosts
+    ziti edge list edge-routers 'isTunnelerEnabled=true'
+    ```
 
-    ziti edge enroll /tmp/hello-host.jwt
+    If none of the routers shown are in your cluster, then refer to [the router deployment guide](/guides/deployments/30-kubernetes/kubernetes-router.mdx) for more information on how to deploy a router.
 
-    ziti edge create config "hello-intercept-config" intercept.v1 \
-        '{"protocols":["tcp"],"addresses":["minihello.ziti"], "portRanges":[{"low":80, "high":80}]}'
+1. Add a role to the router's tunneler identity. This labels the identity so we can give it permission to host the service as a reverse-proxy. Substitute the router name for `router1`.
```

Comment:
I'd change the wording just a bit here. I'd start by saying "after selecting the router from the output of the above command, add a role to the router's identity allowing it to host the demo service. For example, if the router found was named `router1`, the command to issue is shown below

something liek that that just whacks them  over the head a bit more

Reply:
https://github.com/openziti/ziti-doc/pull/874